luxtronik/__init__.py

```diff
@@ -233,3 +252,40 @@ def _read_char(self):
         "Low-level helper to receive a signed int"
         reading = self._socket.recv(LUXTRONIK_SOCKET_READ_SIZE_CHAR)
         return struct.unpack(">b", reading)[0]
+
+
+class Luxtronik(LuxtronikData):
+    def __init__(self, host, port=LUXTRONIK_DEFAULT_PORT, safe=True):
+        super().__init__(Parameters(safe), Calculations(), Visibilities())
+        self.interface = LuxtronikInterface(host, port, safe)
+        self.read()
+
+    def read(self) -> LuxtronikData:
+        """
+        Read data from heat pump.
+        All available data will be read from the heat pump.
+        """
+        self.interface.read(self)
+        return LuxtronikData.copy(self)
```

Comment:
Do we really need to `copy()` here? If somebody wants to have a copy, he can do a copy himself.